src/sentry/search/events/datasets/discover.py

```diff
@@ -256,6 +266,19 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     default_result_type="percentage",
                 ),
+                SnQLFunction(
+                    "group_array",
+                    # setting 10k as arbitraty upper limit for max_size argument, clickhouse limit is 100M
+                    required_args=[NumberRange("max_size", 0, 10000), ColumnTagArg("column")],
```

Comment:
Should we set this to 100 for now since that's currently the only use case?

Reply:
are you suggesting to hardcode the max_size arg or set the upper limit as 100?